kmir/src/kmir/kdist/mir-semantics/rt/data.md

```diff
@@ -1121,52 +1030,6 @@ The arithmetic operations require operands of the same numeric type.
     // infinite precision result must equal truncated result
      andBool truncate(onInt(BOP, ARG1, ARG2), WIDTH, Unsigned) ==Int onInt(BOP, ARG1, ARG2)
     [preserves-definedness]
-
-  // lower-priority rule to catch undefined behaviour
-  rule #compute(
-          BOP,
-          typedValue(Integer(_, WIDTH, SIGNEDNESS), TY, _),
-          typedValue(Integer(_, WIDTH, SIGNEDNESS), TY, _),
-          false) // unchecked
-    => Overflow_U_B
-    requires isArithmetic(BOP)
-    [priority(60)]
-
-  // These are additional high priority rules to detect/report divbyzero and div/rem overflow/underflow
```

Comment:
Similarly we can get rid of `Overflow_U_B` and such now.

Reply:
I think we should consider keeping (or re-adding) these specific rules for division by zero and overflow.
WIth the new code, thunks would be created but we already know this is UB and can detect it.